doc/source/features/strict-mode.rst

```diff
@@ -0,0 +1,263 @@
+.. _strict-mode:
+
+===========
+Strict Mode
+===========
+
+This mode is designed to avoid the use of
+experimental and unsupported features in |PXC|.
+It performs a number of validations at startup and during runtime.
+
+Depending on the actual mode you select,
+upon encountering a failed validation,
+the server will either throw an error
+(halting startup or denying the operation),
+or log a warning and continue running as normal.
+The following modes are available:
+
+* ``DISABLED``: Do not perform strict mode validations
+  and run as normal.
+
+* ``PERMISSIVE``: If a vaidation fails, log a warning and continue running
+  as normal.
+
+* ``ENFORCING``: If a validation fails during startup,
+  halt the server and throw an error.
+  If a validation fails during runtime,
+  deny the operation and throw an error.
+
+* ``MASTER``: The same as ``ENFORCING`` except that the validation of
+  :ref:`explicit table locking <explicit-table-locking>` is not performed.
+  This mode can be used with clusters
+  in which write operations are isolated to a single node.
+
+By default, *enforcing* strict mode is enabled,
+except if the node is acting as a standalone server
+or the node is bootstrapping, then strict mode is disabled.
+
+To select the mode,
+use the :variable:`pxc_strict_mode` variable in the configuration file
+or the ``--pxc-strict-mode`` option during ``mysqld`` startup.
+
+.. note::
+
+   Although it is recommended to start the server with the necessary mode,
```

Comment:
Percona highly recommend user to use pxc_strict_mode = ENFORCING only. If some part of workload fails you may want to relook at your workload as it may be exercising experimental or unsupported feature.